### ISSUE TITLE
supporting measured_by clauses

### ISSUE THREAD

**Issue by JonasKlamroth** [JonasKlamroth/JJBMC#5]
Feature request:
measured_by clauses to show termination of programs with recursive function calls should be supported

**Comment by mattulbrich** [comment]
Are `decreases` clauses in loops supported? Might have a related solution.

**Comment by JonasKlamroth** [comment]
yes they are and i think the solution should work pretty much the same way. just a matter of doing it
